client/src/plugins/usage-statistics/event-handlers/UserTrackedClickEventHandler.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import BaseEventHandler from './BaseEventHandler';
+
+const TRACKED_LINK_PARENT_IDS = [ 'version-info-overlay', 'welcome-page-cloud', 'welcome-page-platform', 'welcome-page-learn-more' ];
+
+/**
+  * Sends event when user clicks a link or button contained within a tracked html container.
+ */
+export default class UserTrackedClickEventHandler extends BaseEventHandler {
```

Comment:
We could simplify the name to `LinkOpenedEventHandler`? Maybe just my opinion but this seems a bit vague since all telemetry events are tracking user actions.